ruby_test.rb

```diff
@@ -0,0 +1,8 @@
+puts "one"
+puts 'two'
+puts 'foo'
+
+foo = "hello"
+bar = "world"
+puts foo +  " " + bar
```

Comment:
Unnecessary spacing detected.<br>Operator `+` should be surrounded with a single space.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.